sound/soc/codecs/max98373.h

```diff
@@ -220,8 +221,8 @@ struct max98373_priv {
 	unsigned int rx_mask;
 };
 
-extern const struct snd_soc_component_driver soc_codec_dev_max98373_sdw;
-extern void max98373_reset(struct max98373_priv *max98373, struct device *dev);
-extern void max98373_slot_config(struct device *dev,
-	struct max98373_priv *max98373);
+const struct snd_soc_component_driver soc_codec_dev_max98373_sdw;
```

Comment:
.. and keep "extern" for soc_codec_dev_max98373_sdw. It is still needed.

Reply:
> Can we also try to remove the make W=1 warning:
> 
> ```
> /data/pbossart/ktest/sof-dev/sound/soc/codecs/max98373-sdw.c: In function ‘max98373_sdw_dai_hw_params’:
> /data/pbossart/ktest/sof-dev/include/linux/bits.h:26:28: warning: comparison of unsigned expression < 0 is always false [-Wtype-limits]
>    26 |   __builtin_constant_p((l) > (h)), (l) > (h), 0)))
>       |                            ^
> /data/pbossart/ktest/sof-dev/include/linux/build_bug.h:16:62: note: in definition of macro ‘BUILD_BUG_ON_ZERO’
>    16 | #define BUILD_BUG_ON_ZERO(e) ((int)(sizeof(struct { int:(-!!(e)); })))
>       |                                                              ^
> /data/pbossart/ktest/sof-dev/include/linux/bits.h:39:3: note: in expansion of macro ‘GENMASK_INPUT_CHECK’
>    39 |  (GENMASK_INPUT_CHECK(h, l) + __GENMASK(h, l))
>       |   ^~~~~~~~~~~~~~~~~~~
> /data/pbossart/ktest/sof-dev/sound/soc/codecs/max98373-sdw.c:627:25: note: in expansion of macro ‘GENMASK’
>   627 |   port_config.ch_mask = GENMASK(stream_config.ch_count - 1, 0);
>       |                         ^~~~~~~
> /data/pbossart/ktest/sof-dev/include/linux/bits.h:26:40: warning: comparison of unsigned expression < 0 is always false [-Wtype-limits]
>    26 |   __builtin_constant_p((l) > (h)), (l) > (h), 0)))
>       |                                        ^
> /data/pbossart/ktest/sof-dev/include/linux/build_bug.h:16:62: note: in definition of macro ‘BUILD_BUG_ON_ZERO’
>    16 | #define BUILD_BUG_ON_ZERO(e) ((int)(sizeof(struct { int:(-!!(e)); })))
>       |                                                              ^
> /data/pbossart/ktest/sof-dev/include/linux/bits.h:39:3: note: in expansion of macro ‘GENMASK_INPUT_CHECK’
>    39 |  (GENMASK_INPUT_CHECK(h, l) + __GENMASK(h, l))
>       |   ^~~~~~~~~~~~~~~~~~~
> /data/pbossart/ktest/sof-dev/sound/soc/codecs/max98373-sdw.c:627:25: note: in expansion of macro ‘GENMASK’
>   627 |   port_config.ch_mask = GENMASK(stream_config.ch_count - 1, 0);
>       |                         ^~~~~~~
> ```
> 
> Thanks!

Shall fix W=1 warning. Thanks!